packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  DisplayOptions,
+  PCD,
+  PCDPackage,
+  SerializedPCD,
+  StringArgument,
+  StringArrayArgument
+} from "@pcd/pcd-types";
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="util/declarations/circomlibjs.d.ts" />
+import { buildEddsa, buildPoseidon } from "circomlibjs";
+import { v4 as uuid } from "uuid";
+import { EdDSACardBody } from "./CardBody";
+
+export const EdDSAPCDTypeName = "eddsa-pcd";
+
+export interface EdDSAPCDArgs {
+  privateKey: StringArgument;
+  message: StringArrayArgument;
+  id: StringArgument;
+}
+
+export interface EdDSAPCDClaim {
+  publicKey: [string, string];
+  message: bigint[];
+}
+
+export interface EdDSAPCDProof {
+  signature: string;
+}
```

Comment:
same here - what is the format of `signature`, and how does `EdDSAPCDProof` relate to `EdDSAPCDClaim`?